app/controllers/exports_controller.rb

```diff
@@ -43,16 +44,28 @@ def locate_labware
   def locate_ancestor_plate
     return nil if export.ancestor_purpose.blank?
 
-    # Store the result of the query in order to support multiple ancestor plates
-    # for the specified ancestor purpose.
-    @ancestor_plate_list = @plate.ancestors.where(purpose_name: export.ancestor_purpose)
-
-    ancestor_result = @ancestor_plate_list.first
+    ancestor_result = @plate.ancestors.where(purpose_name: export.ancestor_purpose).first
     return nil if ancestor_result.blank?
 
     Sequencescape::Api::V2.plate_with_custom_includes(include_parameters, id: ancestor_result.id)
   end
 
+  # Returns an array of all ancestor plates of @plate that match a specific
+  # purpose (defined in the export configuration). If no such plates are found,
+  # an empty array is returned. The result array is made available to the view
+  # that generates the export as @ancestor_plate_list by the show method.
+  #
+  # @return [Array, Sequencescape::Api::V2::Plate] An array of Plate records if
+  #   any are found, otherwise an empty array.
+  def locate_ancestor_plate_list
+    return nil if export.ancestor_purpose.blank?
+
+    # Collect plate ids from Asset resource results to fetch Plates later.
```

Comment:
Can we avoid using the word Asset? 
1. Isn't @plate actually coming from a V2 Plate API query in `locate_labware`?
2. Asset is a historical term from before it was split into Labware and Receptacle, and I'd like to get rid of all references to the word 'Asset' from SS & Limber to avoid confusion.

Reply:
I have updated the method documentation to make it clear.